Getting-Started.md

```diff
@@ -398,6 +399,48 @@ starting/(re)creating/building/attaching containers for a service.
 	```
     And restart the system.
 
+
+#### SLL error [&uarr;](#ssl-)
+
+* Indication: localhost:9200 refuses connection error.
+
+* Diagnosis: 
+
+![ssl_error](https://user-images.githubusercontent.com/53489999/77805355-aea32c00-70a7-11ea-9222-5073851bdc60.png)
```

Comment:
Instead of adding the figure, can you copy the text as similarly done for the links below?
- https://github.com/chaoss/grimoirelab-sirmordred/blob/5e17e8326bf5d07d2b6fc808969b0385297af283/Getting-Started.md#permission-denied-
- https://github.com/chaoss/grimoirelab-sirmordred/blob/5e17e8326bf5d07d2b6fc808969b0385297af283/Getting-Started.md#processes-have-conflicts-with-searchguard-

Thanks